packages/venia-ui/lib/targets/__tests__/venia-ui-targets.spec.js

```diff
@@ -69,6 +72,8 @@ test('uses RichContentRenderers to inject a default strategy into RichContent',
 });
 
 test('uses routes to inject client-routed pages', async () => {
+    jest.setTimeout(WEBPACK_BUILD_TIMEOUT);
```

Comment:
I ensured that this timeout was set for all tests that wait for `buildModuleWith` since that function performs a _full webpack build_ and was intermittently taking longer than the default jest timeout.